test/snapshots/seam-blueprint.test.ts.md

```diff
@@ -184,6 +184,14 @@ Generated by [AVA](https://avajs.dev).
               name: 'system_type_display_name',
               type: 'string',
             },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              isUndocumented: false,
+              name: 'visionline_metadata',
+              type: 'record',
```

Comment:
Where does `record` type comes from? I haven't seen such a type in our openapi spec, only `object`

Reply:
We have RecordProperty and ObjectProperty interfaces. I think a property without more properties nested will be of type record? Don't remember the reasoning behind this but it's unlikely I came up with this distinction myself. Do we want to only maintain ObjectProperty?

Reply:
Oh got it, looks good! I was a bit confused because I thought `record` comes from the openapi spec.

Reply:
Custom metadata is an example of a record type.